src/generic/common/DropdownWithToggle.tsx

```diff
@@ -0,0 +1,79 @@
+import type React from 'react';
```

Comment:
We don't need generic/common folder, we already have shared-components folder. move dropdown component in shared-components folder.